### ISSUE TITLE
File extensions: ".R" vs. ".r"

### ISSUE THREAD

**Issue by SOLV-Code** [avelez-espino/forecastR_phase4#16]
We currently have both versions. Need to pick one.....
(potential (problems)[https://stackoverflow.com/questions/30446679/r-files-in-capital-and-lowercase-difference], and annoying when doing search based on extension)

**Comment by MichaelFolkes** [comment]
I only use .R naming. I assume the .r files are legacy. I recommend we change all to .R as that is, I think, more common.

**Comment by SOLV-Code** [comment]
agreed. Let's stick to that moving forward, and you can fix the legacy stuff as you work through your "reno"

**Comment by MichaelFolkes** [comment]
almost all .r files renamed to .R
 I missed a couple in simplelogpower directory.
the command is:
`git mv filename.r filename.R`
